### PR TITLE
[Finishes #117968685] less margin before videos when topInstructions

### PR DESCRIPTION
**Pull request by Bjvanminnen**
image

There was some space below gameButtons where previously we would have a short form of instructions (that you could click to take you to the long form). We want to get rid of that space when we have instructions in the top pa ne.

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
I started out by trying to move all the gameButtons styling into inline-JSX. This was a fair amount of work (that may need to be done eventually anyways) in part because it meant every app needed to tell the component whether it was in legacyShareView (this would be easier if everything was connectable).

I eventually gave up because I was having trouble matching the behavior of the :after selector in legacy share view, and this was supposed to be a super simple fix.

File: apps/src/templates/GameButtons.jsx [hunk 1]
```diff
@@ -3,6 +3,15 @@ var msg = require('../locale');
 var ProtectedStatefulDiv = require('./ProtectedStatefulDiv');
 var commonStyles = require('../commonStyles');
 
+var styles = {
+  instructionsInTopPane: {
+    // common.scss provides an :after selector that ends up adding 18px of height
+    // to gameButtons. We want to get rid of that when we have don't have
+    // instructions below game buttons
+    marginBottom: -18
```
Comment (Bjvanminnen) [inline apps/src/templates/GameButtons.jsx:11]: Not entirely clear to me why the :after selector takes up 18px, given that it specifies its height and line-height as 0px. As such, I don't love the approach I'm taking here, but it does seem to work.

**Comment by islemaster** [comment]
LGTM.  Applab-only for now, right?

**Comment by Bjvanminnen** [comment]
Good point - should prob do this for game lab too

**Pull request merged**